packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissible.ts

```diff
@@ -22,13 +19,110 @@ export function useInternalChipDismissible({
     chip => !selected.includes(chip.value),
   );
 
+  return { chipOptions, sortedVisibleChipOptions, availableChipOptions };
```

Comment:
nice, I like having this as a function

I think we're mixing arrow functions and `function` functions though. we should try sticking to one syntax.